desktop_app/src/ui/components/Chat/ChatInput/index.tsx

```diff
@@ -88,6 +106,8 @@ export default function ChatInput({
   // Rotating placeholder state
   const [placeholderIndex, setPlaceholderIndex] = useState(0);
 
+  const textareaRef = useRef<HTMLTextAreaElement>(null);
```

Comment:
why do you need to add this ref? Similar question applies to the [`inputRect` prop](https://github.com/archestra-ai/archestra/pull/496/files#diff-1a1ec7c01261e97be3917ebed9d7037193ab402b4943f1eb3a4fccba7ebfb4c9R17) in `desktop_app/src/ui/components/Chat/SlashCommandDropdown/index.tsx`, can we do without that?